dietpi/dietpi-software

```diff
@@ -10830,7 +10830,7 @@ _EOF_
 
 			# Dependencies
 			(( $G_HW_ARCH < 3 )) && G_AGI libatlas3-base # numpy from piwheels deps: https://piwheels.org/project/numpy/
-			(( $G_HW_ARCH == 3 )) && G_AGI gcc # webrtcvad-wheels build on ARMv8: https://pypi.org/project/webrtcvad-wheels/#files
+			(( $G_HW_ARCH == 3 || ( $G_HW_ARCH == 10 && $G_DISTRO > 5 ) )) && G_AGI gcc # webrtcvad-wheels build on ARMv8 and x86_64 Bullseye+: https://pypi.org/project/webrtcvad-wheels/#files
```

Comment:
`x86_64 Bullseye+` without the `+` ?

Reply:
Short form for "Bullseye and above". Could be also "Python >3.8" (to be more precise)?

Reply:
ok got it